starlark/library.go

```diff
@@ -2463,20 +2542,24 @@ func updateDict(dict *Dict, updates Tuple, kwargs []Tuple) error {
 	return nil
 }
 
-func setUpdate(s *Set, args Tuple, kwargs []Tuple) error {
+// forIterableArgsDo is a helper for implementing methods which perform the
+// same operation on an iteratover over each iterable argument, and don't
+// allow keyword arguments - for example, set.update(*others),
+// set.union(*others), etc.
+func forIterableArgsDo(operation func(other Iterator) error, args Tuple, kwargs []Tuple) error {
```

Comment:
It's common style in Go to put the func parameter last so that it can be called like so:
```
forIterableArgsDo(args, kwarks, func() {
     ...
})
```

Of course, the error result makes this slightly less tidy, but I think it's still the preferred order.

Reply:
Thanks for the suggestion, makes the code more readable!